_content/excerpts/study-and-translation-yaksha-samyukta_bingenheimer.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "A Study and Translation of the Yakṣa-saṃyukta in the Shorter Chinese Saṃyukta-āgama"
+authors:
+  - bingenheimer
+editor: dhammadinnā
+external_url: "https://mbingenheimer.net/publications/bingenheimer.2020.yakshaSamyukta.pdf"
+drive_links:
+  - "https://drive.google.com/drive/folders/1-3P5u65MsX2gjuN46xqNV2dJxoAxrQDQ"
+course: cosmology
+tags:
+  - SA
+  - manuscripts
+  - philology
+year: 2020
+oclc: 1225958086
+publisher: "Dharma Drum Publishing Corporation"
+address: "Taiwan"
+volume: 8
+series: "Dharma Drum Institute of Liberal Arts research series"
+booktitle: sa-research_dhammadinna
```

Comment:
Nice, but got them mixed up 😅  The field for linking this is `from_book` not `booktitle`. Also, this file belongs in `papers` not `excerpts` as the book is an edited volume compiling many authors.